backend/tests/reservation/deleteSingle.js

```diff
@@ -0,0 +1,38 @@
+import chai, { assert } from 'chai';
+import chaiHttp from 'chai-http';
+
+chai.use(chaiHttp);
+
+describe('DELETE reservations/single', () => {
+  it('1. should return expected error when reservationID is not provided.', function(done) {
+    chai
+      .request('localhost:3001/reservation')
```

Comment:
we can add a constant for 'localhost:3001/'

Reply:
Yep, its already in the config. Ill swap that out.